File: html/src/org/anyspirit/learning/gdx/client/DragomeConfiguration.java
package org.anyspirit.learning.gdx.client;

import com.dragome.commons.DragomeConfiguratorImplementor;

@DragomeConfiguratorImplementor(priority = 20)
public class DragomeConfiguration extends com.badlogic.gdx.backends.dragome.DragomeConfiguration {

    @Override
    public boolean projectClassPathFilter(String projectPath) {
        boolean include = false;
        include |= projectPath.matches(".*/core-[0-9.]+.jar");
        return include;
    }

    @Override
    public boolean isRemoveUnusedCode() {
        return true;
    }
}

File: html/src/org/anyspirit/learning/gdx/client/DragomeConfiguration.kt
package org.anyspirit.learning.gdx.client

import com.dragome.commons.DragomeConfiguratorImplementor

@DragomeConfiguratorImplementor(priority = 20)
class DragomeConfiguration : com.badlogic.gdx.backends.dragome.DragomeConfiguration() {

    override fun projectClassPathFilter(projectPath: String): Boolean = false or
            ".*/core-[0-9.]+.jar".toRegex().matches(projectPath) or
            ".*/kotlin-stdlib-[0-9.-]+.jar".toRegex().matches(projectPath) or
            ".*/kotlin-runtime-[0-9.-]+.jar".toRegex().matches(projectPath)

    override fun isRemoveUnusedCode(): Boolean = true
}
